lifelines/plotting.py

```diff
@@ -219,7 +219,7 @@ def rmst_plot(model, model2=None, t=np.inf, ax=None, text_position=None, **plot_
     return ax
 
 
-def qq_plot(model, ax=None, **plot_kwargs):
+def qq_plot(model, ax=None, scatter_color="k", **plot_kwargs):
```

Comment:
strange: it seems I don't even use `plot_kwargs`. Probably will be removed in a later version.

Reply:
Oh woah I didnt notice that either, nice call out